index.html

```diff
@@ -907,14 +907,17 @@ <h2>Privacy Considerations</h2>
     <section>
       <h2>Persistent information</h2>
       <p>
-        Since for SVC codecs implemented in WebRTC the use of scalable coding tools
-        is not negotiated and is independent of the supported profiles, and since SVC
-        is rarely supported in hardware encoders, knowledge of the
-        {{RTCRtpEncodingParameters/scalabilityMode}} supported by the {{RTCRtpSender}}
-        does not provide additional information on the underlying hardware.
-        However, since browsers may differ in their support for SVC modes, the supported
-        {{RTCRtpEncodingParameters/scalabilityMode}} may permit differentiation
-        between browsers. This additional fingerprint surface is expected to decrease
+        This specification relies upon [[Media-Capabilities]] for discovery of the
+        values of {{RTCRtpEncodingParameters/scalabilityMode}} supported by each
+        codec. Since this specification does not support capability discovery,
+        only configuration, it does not introduce additional fingerprint surface
+        relating to support for codecs or {{RTCRtpEncodingParameters/scalabilityMode}}
+        values, beyond what is provided by [[Media-Capabilities]].
```

Comment:
I agree with the sentiment, but technically speaking this API is not gated on whether or not MediaCapabilities was queried and permission was granted, so unless we amend that (e.g. "unless MC has given permission to expose configurations, throw an exception when attempting to set scalabilityMode") it is entirely possible to trial-and-error setParameters() (without the use of MediaCapabilities) to try to figure out what is accepted and what results in SW fallback.

Reply:
Or to force fallback to L1T1 unless the desired scalabilityMode has already been exposed

Reply:
WebRTC-SVC does not modify the behavior of WebRTC-PC methods such as `setParameters()`, `addTransceiver` or `createOffer`, none of which are gated on MediaCapabilities.